docs/usage/configuration-options.md

```diff
@@ -2674,7 +2674,9 @@ By default, `renovate` will update to a version greater than `latest` only if th
 
 Must be valid usernames.
 
-If on GitHub and assigning a team to review, you must use the prefix `team:` and add the _last part_ of the team name.
+### Required reviewers on GitHub
```

Comment:
Hmm, adding new headings is forbidden by the **website-docs › has headers for every required sub-option** test:

```diff
expect(received).toEqual(expected) // deep equality

- Expected  - 0
+ Received  + 2

@@ -1,6 +1,8 @@
  Array [
+   "Required reviewers on Azure DevOps",
+   "Required reviewers on GitHub",
    "abortIgnoreStatusCodes",
    "abortOnError",
    "allowedVersions",
    "authType",
    "autoReplaceStringTemplate",

    at Object.<anonymous> (test/website-docs.spec.ts:76:22)
    at runMicrotasks (<anonymous>)
```

What do you want me to do?